src/store/slices/appSlice/imdex.ts

```diff
@@ -0,0 +1 @@
+export * from 'store/slices/appSlice/appSlice';
```

Comment:
you need to rename the file to index.ts. An error was made and imports may not work correctly because of this error